go/test/endtoend/onlineddl/vtgate_util.go

```diff
@@ -206,6 +206,21 @@ func CheckLaunchAllMigrations(t *testing.T, vtParams *mysql.ConnParams, expectCo
 	}
 }
 
+// CheckForceMigrationCutOver marks a migration for forced cut-over, and expects success by counting affected rows
```

Comment:
Nit, but we should use sentence punctuation here and elsewhere.

Reply:
fixed.